test_scripts/Smoke/API/029_OnDriverDistraction_PositiveCase.lua

```diff
@@ -0,0 +1,58 @@
+---------------------------------------------------------------------------------------------------
+-- User story: Smoke
+-- Use case: OnDriverDistraction
+-- Item: Happy path
+--
+-- Requirement summary:
+-- [OnDriverDistraction] SUCCESS: getting SUCCESS:UI.OnDriverDistraction()
+--
+-- Description:
+-- Mobile application sends valid OnDriverDistraction request and gets UI.OnDriverDistraction "SUCCESS"
+-- response from HMI
+
+-- Pre-conditions:
+-- a. HMI and SDL are started
+-- b. appID is registered and activated on SDL
+-- c. appID is currently in Background, Full or Limited HMI level
+
+-- Steps:
+-- appID requests OnDriverDistraction with valid parameters
```

Comment:
@GetmanetsIrina this is not a request, just notification from HMI side triggered by some event, so appID request nothing in this case. Here should be something like:
"HMI sends OnDriverDistraction notification with valid parameters to SDL"